pyproject.toml

```diff
@@ -23,6 +23,7 @@ tabulate = { extras = ["widechars"], version = "0.9.0" }
 jishaku = "2.5.2"
 sentry-sdk = "2.3.1"
 humanize = "4.9.0"
+wand = "^0.6.13"
```

Comment:
For consistency we should either make this an exact version, or update the others to be carot versions.